library/Freckle/App/Stats.hs

```diff
@@ -18,6 +18,9 @@ module Freckle.App.Stats
   , Gauge
   , dbConnections
   , withGauge
+  , lookupGauge
+  , incGauge
+  , decGauge
```

Comment:
Sad that we now have to expose the ability to get this wrong, but I guess there's no way around it.

Reply:
Yeah I was thinking about this. There _might_ be a way to do similar to what youu suggested in a thread somewhere, but we'd need to shuffle a lot of code around/change the typeclass restrictions on certain functions. I dunno.